exwm-xsettings.el

```diff
@@ -181,6 +181,21 @@ These settings take precedence over `exwm-xsettings-theme' and
    (owner :initarg :owner :type xcb:WINDOW))      ;new slot
   :documentation "An XSETTINGS client message.")
 
+(eval-and-compile
+  (fset 'exwm-xsettings--color-dark-p
+        (if (< emacs-major-version 29)
+            ;; Borrowed from Emacs 29.
+            (lambda (rgb)
+              "Whether RGB is more readable against white than black."
+              (unless (<= 0 (apply #'min rgb) (apply #'max rgb) 1)
+                (error "RGB components %S not in [0,1]" rgb))
+              (let* ((r (expt (nth 0 rgb) 2.2))
+                     (g (expt (nth 1 rgb) 2.2))
+                     (b (expt (nth 2 rgb) 2.2))
+                     (y (+ (* r 0.2126) (* g 0.7152) (* b 0.0722))))
+                (< y 0.325)))
```

Comment:
I shrunk this code down a bit and inlined the constant (we're not maintaining this code here, so readability isn't a goal.

Reply:
Inlining the constant makes sense.